packages/cozy-harvest-lib/src/components/cards/LaunchTriggerCard.jsx

```diff
@@ -60,11 +60,6 @@ export class LaunchTriggerCard extends Component {
                   </li>
                 </ul>
                 <div>
-                  {error && (
-                    <Text className="u-error u-mb-1">
-                      {t('card.launchTrigger.error')}
-                    </Text>
-                  )}
```

Comment:
Does that mean that we already display the error somewhere? If yes, where?

Reply:
Yes, it's [here](https://github.com/cozy/cozy-libs/blob/master/packages/cozy-harvest-lib/src/components/KonnectorConfiguration/KonnectorAccountTabs.jsx#L89-L91) — just above this component :)